src/main.cc

```diff
@@ -85,20 +85,26 @@ static int daemon_main(LiteClient& client, const bpo::variables_map& variables_m
   return run_daemon(client, interval, return_on_sleep, true);
 }
 
-static int cli_install(LiteClient& client, const bpo::variables_map& params) {
-  // Make sure no other update instances are running, i.e. neither the daemon or other CLI update/finalize
-  std::string target_name;
-  int version = -1;
+static void get_target_id(const bpo::variables_map& params, int& version, std::string& name) {
+  name = "";
+  version = -1;
   if (params.count("update-name") > 0) {
     const auto version_str = params["update-name"].as<std::string>();
     try {
       version = std::stoi(version_str);
     } catch (const std::invalid_argument& exc) {
       LOG_INFO << "Failed to convert the input target version to integer, consider it as a name of Target: "
```

Comment:
I don't know why I made the log as "info" log in the first place. Now, it looks like it makes sense to turn it into a debug or even trace error.

Reply:
It caught my attention too. Specially because the parameter has "name" in it. I've added a new commit changing the log to LOG_DEBUG.

Reply:
Thanks.